common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
This CoA could use more space.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122811641-cb95d180-d2e1-11eb-8fae-6ce3e98a2438.png)

</details>